lsp-mode.el

```diff
@@ -4255,9 +4257,9 @@ Also, additional data to attached to each candidate can be passed via PLIST."
                                                 item)
                                        item))))
                    (prefix-line (buffer-substring-no-properties (point-at-bol) (point))))
-             (setf done? (or (seqp resp)
-                             (not (gethash "isIncomplete" resp)))
-                   lsp--capf-cache (when (and done? (not (seq-empty-p items)))
+             (setf done? (or (seqp resp) (not (gethash "isIncomplete" resp)))
+                   lsp--capf-cache (when (and done? (not (seq-empty-p items))
+                                              (not (equal bounds-start (point))))
```

Comment:
can you elaborate on that? What will happen if I do not have a prefix and manually invoke the completion? Will it reuse the cache after the first invocation?

Reply:
No, hmm think again this one is not really needed, I will remove this.